lib/omnibus/build_version_dsl.rb

```diff
@@ -117,8 +117,11 @@ def build_version=(new_version)
     # @param [String] version
     # @return [String]
     def maybe_append_timestamp(version)
+      log.info(log_key) { "class Omnibus::BuildVersionDSL - maybe_append_timestamp method: Config.append_timestamp: #{Config.append_timestamp}" }
+      log.info(log_key) { "class Omnibus::BuildVersionDSL - maybe_append_timestamp method: version: #{version}" }
+      log.info(log_key) { "class Omnibus::BuildVersionDSL - maybe_append_timestamp method: has_timestamp?(version): #{has_timestamp?(version)}" }
       if Config.append_timestamp && !has_timestamp?(version)
-        [version, Omnibus::BuildVersion.build_start_time].join("+")
+        [version, Omnibus::BuildVersion.build_start_time].join("-")
```

Comment:
This was to address an issue where the signtool wasnt able to sign the file with a `+` in the name. It would rename the file to a `;` then we were getting an error where the _file was not found_